recipes/_environment.rb

```diff
@@ -110,6 +110,10 @@
   end
 else
 
+  if aix?
+    omnibus_env['PATH'].unshift('/opt/IBM/xlC/13.1.0/bin:/opt/IBM/xlC/13.1.0/bin')
+  end
+
```

Comment:
the XLC paths need to appear before `/opt/#{node['omnibus']['toolchain_name']}/embedded/bin` in PATH. Similarly, `/usr/sfw|ccs/bin` need to appear after.